spec/rubocop/cop/internal_affairs/node_type_predicate_spec.rb

```diff
@@ -3,11 +3,19 @@
 describe RuboCop::Cop::InternalAffairs::NodeTypePredicate do
   subject(:cop) { described_class.new }
 
-  it 'registers an offense for a comparison node type check' do
-    expect_offense(<<-RUBY, 'example_cop.rb')
+  context 'comparison node type check' do
+    it 'registers an offense' do
+      expect_offense(<<-RUBY, 'example_cop.rb')
       node.type == :send
       ^^^^^^^^^^^^^^^^^^ Use `#send_type?` to check node type.
-    RUBY
+      RUBY
+    end
+
+    it 'corrects' do
```

Comment:
In the other commit you've used the description "auto-corrects", which I prefer.